backend/src/auth/CustomJwtPayload.ts

```diff
@@ -1,5 +1,5 @@
 import { JwtPayload } from 'jsonwebtoken'
 
 export interface CustomJwtPayload extends JwtPayload {
-  pubKey: Buffer
+  gradidoID: Buffer
```

Comment:
is there a reason you use a buffer? Why not just use plain text? does the jwt interface require a buffer?